kafka/pom.xml

```diff
@@ -198,7 +198,7 @@
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka_2.11</artifactId>
-      <version>0.9.0.0</version>
```

Comment:
The operator is supposed to work with any 0.9.x.x kafka brokers and clients unless the 0.9.0.0 API can not meet our requirement.